authors/settings.py

```diff
@@ -80,8 +80,12 @@
 
 DATABASES = {
```

Comment:
All the values are hard coded here, that is a security risk, instead use environment variables.

Reply:
@inno-asiimwe that's now covered with environments variables, new changes to be pushed.